rllib/core/learner/torch/torch_learner.py

```diff
@@ -156,7 +156,9 @@ def _uncompiled_update(
         with contextlib.ExitStack() as stack:
             if self.config.num_learners > 1:
                 for mod in self.module.values():
-                    stack.enter_context(mod.no_sync())
+                    # Skip non-torch modules, b/c they may not have the `no_sync` API.
+                    if isinstance(mod, torch.nn.Module):
```

Comment:
Dumb question: What kind of modules are no torch modules?

Reply:
For example: `RandomRLModule` often used in self-play setups.